app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt

```diff
@@ -61,23 +59,18 @@ class LandingPageFragment : Fragment() {
         }
     }
 
-    private fun filterLaunches(searchTerm: String?) {
-        adapter.filter(searchTerm?.lowercase())
-        setListHeaderText(adapter.itemCount)
-    }
-
     private fun navigateToLaunchDetails(launch: Launch) {
         val bundle = Bundle()
         bundle.apply {
-            putString(LaunchDetailBundle.id, launch.id)
-            putString(LaunchDetailBundle.missionName, launch.mission_name)
-            putString(LaunchDetailBundle.launchSite, launch.launch_site_long)
-            putString(LaunchDetailBundle.launchDate, launch.launch_date_utc)
-            putString(LaunchDetailBundle.launchYear, launch.launch_year)
-            putString(LaunchDetailBundle.details, launch.details)
-            putString(LaunchDetailBundle.articleLink, launch.article_link)
-            putString(LaunchDetailBundle.videoLink, launch.video_link)
-            putStringArray(LaunchDetailBundle.imageLinks, launch.image_links?.toTypedArray())
+            putString(LaunchDetailFields.id, launch.id)
+            putString(LaunchDetailFields.missionName, launch.mission_name)
+            putString(LaunchDetailFields.launchSite, launch.launch_site_long)
+            putString(LaunchDetailFields.launchDate, launch.launch_date_utc)
+            putString(LaunchDetailFields.launchYear, launch.launch_year)
+            putString(LaunchDetailFields.details, launch.details)
+            putString(LaunchDetailFields.articleLink, launch.article_link)
+            putString(LaunchDetailFields.videoLink, launch.video_link)
+            putStringArray(LaunchDetailFields.imageLinks, launch.image_links?.toTypedArray())
```

Comment:
I think you could simplify this with `let` so you don't have to keep repeating LaunchDetailFields.

```
bundle.apply {
    LaunchDetailBundle.let {
        putString(it.id, launch.id)
        ...
}

```

Reply:
Good idea, I always have to remind myself that this exists.